diff --git a/repos/spack_repo/builtin/packages/feq_parse/package.py b/repos/spack_repo/builtin/packages/feq_parse/package.py
index e4b960b7..bc0916b9 100644
--- a/repos/spack_repo/builtin/packages/feq_parse/package.py
+++ b/repos/spack_repo/builtin/packages/feq_parse/package.py
@@ -29,6 +29,7 @@ class FeqParse(CMakePackage):
     version("1.0.2", sha256="1cd1db7562908ea16fc65dc5268b654405d0b3d9dcfe11f409949c431b48a3e8")
 
     depends_on("fortran", type="build")  # generated
+    depends_on("c", type="build")  # generated
 
     depends_on("cmake@3.0.2:", type="build")
 
